modules/data.atmosphere/R/download.NOAA_GEFS_downscale.R

```diff
@@ -0,0 +1,341 @@
+  
+  
+  joined<-  dplyr::inner_join(gefs_hour, nonSW.flux.hrly, by = c("NOAA.member", "timestamp"))
+  
+  joined <- dplyr::inner_join(joined, ShortWave.ds, by = c("NOAA.member", "timestamp")) %>% 
+    dplyr::distinct() %>% 
+    dplyr::mutate(surface_downwelling_shortwave_flux_in_air = ifelse(surface_downwelling_shortwave_flux_in_air < 0, 0, surface_downwelling_shortwave_flux_in_air),
+                  specific_humidity = ifelse(specific_humidity <0, 0, specific_humidity),
+                  air_temperature = ifelse(air_temperature > 320, NA, air_temperature),
+                  air_temperature = ifelse(air_temperature < 240, NA, air_temperature),
+                  precipitation_flux = ifelse(precipitation_flux < 0, 0, precipitation_flux),
+                  surface_downwelling_longwave_flux_in_air = ifelse(surface_downwelling_longwave_flux_in_air < 0, NA, surface_downwelling_longwave_flux_in_air),
+                  wind_speed = ifelse(wind_speed <0, 0, wind_speed)) %>%
+    dplyr::filter(is.na(timestamp) == FALSE)
+  
+  
+  
+  
+  #############################################
+  # Done with data processing.  Now writing the data to the specified directory. Each ensemble member is written to its own file, for a total
+  # of 21 files.  
+  if (!dir.exists(outfolder)) {
+    dir.create(outfolder, recursive=TRUE, showWarnings = FALSE)
+  }
+  
+  # Create a data frame with information about the file.  This data frame's format is an internal PEcAn standard, and is stored in the BETY database to
+  # locate the data file.  The data file is stored on the local machine where the download occured.  Because NOAA GEFS is an 
+  # ensemble of 21 different forecast models, each model gets its own data frame.  All of the information is the same for 
+  # each file except for the file name.
+  results = data.frame(
+    file = "",                            #Path to the file (added in loop below).
+    host = PEcAn.remote::fqdn(),          #Name of the server where the file is stored
+    mimetype = "application/x-netcdf",    #Format the data is saved in
+    formatname = "CF Meteorology",        #Type of data
+    startdate = paste0(format(start_date, "%Y-%m-%dT%H:%M:00")),    #starting date and time, down to the second
+    enddate = paste0(format(end_date, "%Y-%m-%dT%H:%M:00")),        #ending date and time, down to the second
+    dbfile.name = "NOAA_GEFS_downscale",            #Source of data (ensemble number will be added later)
+    stringsAsFactors = FALSE
+  )
+  
+  results_list = list()
+  
+  #Each ensemble gets its own file.
+  #These dimensions will be used for all 21 ncdf4 file members, so they're all declared once here.
+  #The data is really one-dimensional for each file (though we include lattitude and longitude dimensions
+  #to comply with the PEcAn standard).
+  time_dim = ncdf4::ncdim_def(name="time", 
+                              paste(units="hours since", format(start_date, "%Y-%m-%dT%H:%M")), 
```

Comment:
I think this is a typo?

```suggestion
                              units = paste("hours since", format(start_date, "%Y-%m-%dT%H:%M")), 
```